### PR TITLE
javascript-carousel | Rich Trunzo

### PR DESCRIPTION
**Pull request by richtrunzo**

### CODE REVIEW

File: javascript-carousel/index.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        <div class="circle">
+          <i class="far fa-circle circle-btn circle-btn-one"></i>
+          <i class="far fa-circle circle-btn circle-btn-two"></i>
+          <i class="far fa-circle circle-btn circle-btn-three"></i>
+          <i class="far fa-circle circle-btn circle-btn-four"></i>
+          <i class="fas fa-circle circle-btn circle-btn-five"></i>
+        </div>
+      </div>
+      <i class="fas fa-chevron-right arrow-button arrow-btn-right"></i>
+    </div>
```
Comment (shadorki) [inline javascript-carousel/index.html]: You dont want 5 carousels that you switch between, you want one carousel with 5 images you switch between.

**Review by shadorki — CHANGES_REQUESTED** [review]

File: javascript-carousel/index.html (hunk 1 above)
Comment (richtrunzo) [reply javascript-carousel/index.html]: Oh got it! Okay I'll make that change.

Comment (richtrunzo) [reply javascript-carousel/index.html]: New gif for the updated version:

File: javascript-carousel/main.js [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+    $squirtle.className = "squirtle hidden";
+    $pikachu.className = "pikachu";
+    $jigglypuff.className = "jigglypuff hidden"
+    $circleOne.className = "far fa-circle circle-btn circle-btn-one";
+    $circleTwo.className = "far fa-circle circle-btn circle-btn-two";
+    $circleThree.className = "far fa-circle circle-btn circle-btn-three";
+    $circleFour.className = "fas fa-circle circle-btn circle-btn-four";
+    $circleFive.className = "far fa-circle circle-btn circle-btn-five";
+  }
+})
```
Comment (shadorki) [inline javascript-carousel/main.js]: This logic can be minified using a loop and some sort of internal logic in your code.  Keep track of the position of the images in a global variable so you don't ask the DOM what is currently happening.

File: javascript-carousel/main.js [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+    $charmander.className = "charmander hidden";
+    $squirtle.className = "squirtle hidden";
+    $pikachu.className = "pikachu hidden";
+    $jigglypuff.className = "jigglypuff hidden"
+    $circleOne.className = "fas fa-circle circle-btn circle-btn-one";
+    $circleTwo.className = "far fa-circle circle-btn circle-btn-two";
+    $circleThree.className = "far fa-circle circle-btn circle-btn-three";
+    $circleFour.className = "far fa-circle circle-btn circle-btn-four";
+    $circleFive.className = "far fa-circle circle-btn circle-btn-five";
+  }
```
Comment (shadorki) [inline javascript-carousel/main.js]: Same goes for here, you can iterate through a loop of dom elements, and verify what should be on the dom using a variable in your code.

**Review by shadorki — CHANGES_REQUESTED** [review]

File: javascript-carousel/main.js (hunk 3 above)
Comment (richtrunzo) [reply javascript-carousel/main.js]: Updated per the feedback above, new gif is below.

**Review by BrettAlbright — APPROVED** [review]
👍

**Pull request merged**